requirements/requirements.in

```diff
@@ -22,6 +22,7 @@ ghdiff==0.4
 gunicorn
 lxml~=4.3.0
 kafka-python~=1.4
+kombu==4.2.2.post1  # Remove once fix is released for https://github.com/celery/kombu/issues/1006
```

Comment:
Can you change to `kombu!=4.3.0` in this? I believe that is the version that was broken, and 4.4.0 was released, but it requires a more recent version of redis than we can support due to other dependencies. I think if using the earlier kombu blacklist, none of the other files should change for now, but would allow us to upgrade once we can upgrade our redis implementation

Reply:
I'm reluctant to upgrade to latest ```kombu``` because of this issue: https://github.com/celery/kombu/issues/1016#issuecomment-469319729

It looks like we would have to upgrade ```redis``` before using 4.4.0.

Reply:
Sorry, didn't parse everything you said.  Re-reading before I post again...

Reply:
Updated: https://github.com/dimagi/commcare-hq/pull/23627/commits/5ab6c6806d2b19f5fdb63fdbd6587200e60b99ae